mne/viz/raw.py

```diff
@@ -665,12 +671,14 @@ def plot_raw_psds(raw, tmin=0.0, tmax=60.0, fmin=0, fmax=np.inf,
 
     for ii, (picks, title, ax) in enumerate(zip(picks_list, titles_list,
                                                 ax_list)):
-        psds, freqs = compute_raw_psd(raw, tmin=tmin, tmax=tmax, picks=picks,
-                                      fmin=fmin, fmax=fmax, n_fft=n_fft,
-                                      n_jobs=n_jobs, plot=False, proj=proj)
+        psds, freqs = compute_raw_psd(raw, tmin=tmin, tmax=tmax, fmin=fmin,
+                                      fmax=fmax, proj=proj, n_fft=n_fft,
+                                      picks=picks, window_size=window_size,
+                                      n_overlap=n_overlap, n_jobs=1,
+                                      verbose=None)
 
         # Convert PSDs to dB
-        psds = 10 * np.log10(psds)
+        psds = 20 * np.log10(psds)
```

Comment:
oups my bad if psds is power (not amplitude) than it should be 10 * np.log10

please revert here and change above.